eventuals/scheduler.h

```diff
@@ -71,21 +103,44 @@ class Scheduler {
       }
     }
 
-    auto* scheduler() {
-      return scheduler_;
+    Context Clone() {
+      Context child(scheduler_, "cloned", nullptr, this);
+
+      child.scheduler()->Clone(child, *this);
+
+      return child;
     }
 
+    // There is the most common set of variables to create contexts.
+    bool blocked_ = false;
```

Comment:
Let's actually change this up a bit. Let's make `blocked_` be private and have a "setter" called `block()` that sets `blocked_` to `true` and a getter called `blocked()` that returns whether or not it's blocked.